pkg/controller/gitjob/gitjobs_test.go

```diff
@@ -47,10 +51,13 @@ var _ = Describe("Gitjob Controller", func() {
 		jobmock = mocks.NewMockJobClient(ctrl)
 		gitjobmock = mocks.NewMockGitJobController(ctrl)
 		gitjobmock.EXPECT().EnqueueAfter(gomock.Any(), gomock.Any(), gomock.Any())
+		secretmock = mocks.NewMockSecretCache(ctrl)
+		secretmock.EXPECT().Get(gomock.Any(), gomock.Any()).Return(nil, errors.NewNotFound(schema.GroupResource{}, "not found"))
```

Comment:
yes we don't need that anymore because of [these](https://github.com/rancher/gitjob/commit/01fee94a68dd703fcf08c8ecad6e9b6443b9cb77) changes, removed [here](https://github.com/rancher/gitjob/pull/302/commits/0a14eadef9bb751afb35dd69a85a45787471e486)